recipes/diligentgraphics-volk/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "cci.20211005":
```

Seems to have released tags?

```suggestion
  "1.2.190":
```

---

I see lower it need to match for the dependency... Maybe we can make a object to map this to it's dep requirement

---

There is already a volk Conan recipe. I only need to make it work with diligentgraphics-vulkan-headers.

---

I apologize I am not familiar with these libraries.

The naming on the other 3 you have added makes sense since the are authored DiligentGraphics... but here you are only changing the package.

I'd of expect package overrides to work in this case :thinking: 

The original expects

https://github.com/conan-io/conan-center-index/blob/d1884f5e2a303d95c0e625832d4c037bd5bc914c/recipes/volk/all/conanfile.py#L45

Which should be satisfied by

https://github.com/conan-io/conan-center-index/blob/d1884f5e2a303d95c0e625832d4c037bd5bc914c/recipes/diligentgraphics-vulkan-headers/all/conanfile.py#L15

---

:wave: @madebr @SpaceIm Can you take a peek at this PR please?